yw_website/yw_website/apps/website/program_blocks.py

```diff
@@ -0,0 +1,11 @@
+class ProgramBlocks:
+
+    def __init__(self):
+        self.parent = None
+        self.name = None
+        self.programblock_id = None
+        self.id = None
+        self.in_program_block_id = None
```

Comment:
1. we're going to want a way to reference children, so there needs to be a field with a list of all direct descendants
2. the parent probably isn't too important from the perspective of the UI, so we can probably omit that to simplify the logic